Gemfile

```diff
@@ -1,8 +1,11 @@
 source 'https://rubygems.org'
 gemspec
 
+gem 'actionview'
 gem 'activerecord'
 
+gem 'nokogiri', '~> 1.6.0' if RUBY_VERSION.start_with?('2.0')
```

Comment:
Why is this necessary?

Also, maybe it's time to drop support for Ruby 2.0 given that it has been EOLed for a while.